renderdoc/driver/gl/official/eglplatform.h

```diff
@@ -95,6 +95,8 @@ typedef void*                           EGLNativeDisplayType;
 
 #elif defined(__unix__)
 
+#if ENABLED(RDOC_XLIB)
```

Comment:
This is a system header and should not be modified.

Reply:
This is pretty much the same as how the types are defined in https://github.com/KhronosGroup/EGL-Registry/blob/main/api/EGL/eglplatform.h.